src/feature/feature_utils.cairo

```diff
@@ -15,13 +16,21 @@ use satoru::data::data_store::{IDataStoreSafeDispatcher, IDataStoreSafeDispatche
 /// # Returns
 /// whether the feature is disabled.
 fn is_feature_disabled(data_store: IDataStoreSafeDispatcher, key: felt252) -> bool {
-    // TODO
-    true
+    let response = match data_store.get_bool(key) {
+        Result::Ok(v) => v,
+        Result::Err => Option::None
+    };
+
+    match response {
+        Option::Some(v) => !v,
```

Comment:
Shouldn't we return directly v and not !v

Reply:
Uhmm i thought like, it should return true if feature disabled or not exist from function name. Am I wrong?